e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -640,13 +640,13 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
         });
 
       // We're getting an input element which is 0x0 in size
-      cy.findByLabelText("Transparent").click({ force: true });
+      cy.findByLabelText("Dark").click({ force: true });
```

Comment:
Fix the test because `transparent` is not selectable anymore, we just want to select any theme here expect `light`